src/main/java/com/linecorp/armeria/server/http/jetty/JettyServiceInvocationHandler.java

```diff
@@ -54,17 +56,24 @@
 
 final class JettyServiceInvocationHandler implements ServiceInvocationHandler {
 
-    private final String hostname;
+    private static final Logger logger = LoggerFactory.getLogger(JettyServiceInvocationHandler.class);
+
     private final Function<ExecutorService, Server> serverFactory;
-    private final Configurator configurator = new Configurator();
+    private final Configurator configurator;
+    private final boolean destroyOnStop;
+
+    private String hostname;
     private Server server;
     private ArmeriaConnector connector;
     private com.linecorp.armeria.server.Server armeriaServer;
     private boolean startedServer;
 
-    JettyServiceInvocationHandler(String hostname, Function<ExecutorService, Server> serverFactory) {
+    JettyServiceInvocationHandler(String hostname, Function<ExecutorService, Server> serverFactory,
+                                  boolean destroyOnStop) {
```

Comment:
Replaced `destroyOnStop` with `postStopTask` (`Consumer`) for both Jetty and Tomcat.